docs/onboarding/06-e2e-sample.md

```diff
@@ -0,0 +1,35 @@
+# Chapter 6: Deploy and run end-to-end sample
+
+The following docs will introduce an end-to-end solution where an Azure Digital Twins instance is connected to other Azure services for management of devices and data.
+
+## Key Points
+
+- Create an end-to-end scenario that shows Azure Digital Twins being driven by live device data.
```

Comment:
Rather than a description of what the reader will be doing as part of this challenge, use this section to summarize key lessons should understand before moving forward into the challenge. For example...
- Digital twins are created and updated in an ingestion pipeline that sources data from real-world devices.
- Azure Functions can facilitate the transformation of device data to digital twin properties and telemetry

Reply:
Adjusted, but will leave comment unresolved until re-review